tests/client_tests/rust/Cargo.lock

```diff
@@ -779,7 +783,16 @@ version = "0.48.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "677d2418bec65e3338edb076e806bc1ec15693c5d0104683f2efe857f61056a9"
 dependencies = [
- "windows-targets",
+ "windows-targets 0.48.5",
+]
+
+[[package]]
+name = "windows-sys"
+version = "0.52.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "282be5f36a8ce781fad8c8ae18fa3f9beff57ec1b52cb3de0789201425d9a33d"
+dependencies = [
+ "windows-targets 0.52.4",
```

Comment:
This section duplicates the section above, shouldn't the previous be removed?
The same applies for all the `windows-*` sections from here and below.

Reply:
IDK. I just invoked `cargo update` and presented the outcome here.

Reply:
I've adjusted the Cargo.lock file manually with 030ca1c5d0793f5900. Apparently, the job still succeeds, so we can pretend the changes are sound?

https://jenkins.crate.io/blue/organizations/jenkins/CrateDB%2Fqa%2Fcrate_qa_on_pr/detail/crate_qa_on_pr/380/pipeline/42